build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -23,16 +22,6 @@ jobs:
     - ${{ parameters.dependsOn }}
   condition: ${{ parameters.condition }}
 
-  # XES tasks need to be on PackageES machines.
```

Comment:
:-)

Reply:
I don't think we need this task anymore for ESRP signing?

Reply:
I think because we are using ESRP now instead of PackageES codesign we don't need to be on packageES pool.

Reply:
Yea I removed that but for some reason the CreateNugetPackage job refuse to run. 
https://dev.azure.com/microsoft/ProjectReunion/_build/results?buildId=28785553&view=logs&j=71b158ba-2b04-598f-7447-b2e90984d12a

I thought maybe the yml file was incorrect so it wouldn't run, so I added it back in to run again.